### ISSUE TITLE
"audiohealth power" croaks on specific audio files

### ISSUE THREAD

**Issue by amotl** [hiveeyes/audiohealth#5]
Hi there,

initially reported by @Jodaille at [1] and just recently also by Paul at [2], invoking `audiohealth power` croaks like
```python
Traceback (most recent call last):
  File "/home/pi/audiohealth/.venv27/bin/audiohealth", line 11, in <module>
    load_entry_point('audiohealth', 'console_scripts', 'audiohealth')()
  File "/home/pi/audiohealth/audiohealth.py", line 488, in main
    tmpfile   = power_spectrum(wavfile)
  File "/home/pi/audiohealth/audiohealth.py", line 398, in power_spectrum
    freq500 = max(band500, key=peak_data.get)
ValueError: max() arg is an empty sequence
```

The problem can be reproduced like this:
```shell
$ wget -O PCM004.WAV \
  https://community.hiveeyes.org/uploads/default/original/2X/7/7595f1ad65c69e20c87dcb324f5d049080315aef.wav
audiohealth power --audiofile=PCM004.WAV --pngfile=power.png
```

```shell
$ file PCM004.WAV
PCM004.WAV: RIFF (little-endian) data, WAVE audio
```

With kind regards,
Andreas.

P.S.: Thanks again, Paul, for providing those example files.

[1] https://community.hiveeyes.org/t/how-to-clone-install-and-run-the-audiohealth-program/430/13
[2] https://community.hiveeyes.org/t/how-to-clone-install-and-run-the-audiohealth-program/430/44
[3] https://community.hiveeyes.org/t/how-to-clone-install-and-run-the-audiohealth-program/430/52

**Comment by amotl** [comment]
On the other hand, when now trying with @Jodaille's test file, `audiohealth power` also croaks, but differently:
```shell
wget https://jodaille.org/beehive_audio_monitor/test_audiohealth.wav
$ audiohealth power --audiofile=test_audiohealth.wav --pngfile=power.png
```
```python
Traceback (most recent call last):
  File "/Users/amo/dev/hiveeyes/sources/audiohealth/.venv/bin/audiohealth", line 33, in <module>
    sys.exit(load_entry_point('audiohealth', 'console_scripts', 'audiohealth')())
  File "/Users/amo/dev/hiveeyes/sources/audiohealth/audiohealth.py", line 517, in main
    tmpfile   = power_spectrum(wavfile)
  File "/Users/amo/dev/hiveeyes/sources/audiohealth/audiohealth.py", line 338, in power_spectrum
    x = np.swapaxes(x, 0, 1)
  File "<__array_function__ internals>", line 5, in swapaxes
  File "/usr/local/lib/python3.9/site-packages/numpy/core/fromnumeric.py", line 594, in swapaxes
    return _wrapfunc(a, 'swapaxes', axis1, axis2)
  File "/usr/local/lib/python3.9/site-packages/numpy/core/fromnumeric.py", line 57, in _wrapfunc
    return bound(*args, **kwds)
numpy.AxisError: axis2: axis 1 is out of bounds for array of dimension 1
```
```shell
$ file test_audiohealth.wav
test_audiohealth.wav: RIFF (little-endian) data, WAVE audio, Microsoft PCM, 16 bit, mono 44100 Hz
```

**Comment by amotl** [comment]
The errors have been fixed with 7de5c0426 and efed615fc9.
